OpenOSP Docker fails to install fully

Problem:  Running ./openosp bootstrap fails at installing the FaxWs (error copied below)

Expected: All Docker containers should install successfully OR an option to install the FaxWs should be available when initiating bootstrap

Error: 
```
Step 1/11 : FROM tomcat:8.5-jre
manifest for tomcat:8.5-jre not found: manifest unknown: manifest unknown
ERROR: Service 'faxws' failed to build
Error: No such container:path: cedarwood_faxws_1:/create_database.sql
bash: _bootstrap.sql: No such file or directory
ERROR: 1
rm: cannot remove '_bootstrap.sql': No such file or directory
Setting authentication database...
ERROR 1146 (42S02) at line 1: Table 'OscarFax.users' doesn't exist
ERROR: 1
ERROR 1146 (42S02) at line 1: Table 'OscarFax.user_roles' doesn't exist
ERROR: 1
```

Problem 2:  Expedius fails to launch after running ./openosp start  (error below)

Expected:  Expedius should launch

Expedius launch error:

```
Sending build context to Docker daemon  17.41kB

Step 1/5 : FROM tomcat:8.5-jre
manifest for tomcat:8.5-jre not found: manifest unknown: manifest unknown
ERROR: Service 'expedius' failed to build
```

Has there been any more thoughts or progress around this? I'm trying to solve this myself, but I'm a bit out of my normal stack here.

After having installed maven on my machine (macOS Big Sur 11.3), and running the build `./bin/build-faxws.sh` script I get a lot farther, but error out at 

```zsh
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-war-plugin:2.2:war (default-war) on project FaxWs: Execution default-war of goal org.apache.maven.plugins:maven-war-plugin:2.2:war failed: Unable to load the mojo 'war' in the plugin 'org.apache.maven.plugins:maven-war-plugin:2.2' due to an API incompatibility: org.codehaus.plexus.component.repository.exception.ComponentLookupException: Cannot access defaults field of Properties
```

--- a/bin/setup-faxws.sh
+++ b/bin/setup-faxws.sh
@@ -27,7 +27,7 @@ docker-compose exec db mysql -h localhost -uroot -p$MYSQL_ROOT_PASSWORD -e "crea
 # [cvo] use sql bootstrap instead for now.
 #docker-compose exec faxws bash -c "cd /usr/local/tomcat/webapps/faxWs/META-INF/maven/oscarFax/FaxWs && mvn flyway:migrate -Dflyway.user=root -Dflyway.password=tzcbU/u87wM= -Dflyway.url=jdbc:mysql://db/oscarFax"
 dcid=$(pwd | grep -oh "[^/]*$")
-docker cp ${dcid}_faxws_1:/create_database.sql _bootstrap.sql
+docker cp ${dcid}-faxws-1:/create_database.sql _bootstrap.sql
 
 docker-compose exec db bash -c "mysql -h localhost -uroot -p$MYSQL_ROOT_PASSWORD OscarFax < _bootstrap.sql"
 rm _bootstrap.sql

